### DISCUSSION TITLE
Configurable window borders

### DISCUSSION THREAD

**Discussion by dmitry-erin** [labwc/labwc#1092] [Ideas]
Hi!
The configurable window's borders could be a good idea. 
Does it require significant effort to add, for example, border color to windows rules? So that it would be possible to customize border for one particular app.
Thanks.

**Comment by johanmalm** [comment]
Hi there.
Yes, significant effort to do it well.
I'm happy to merge such a feature if someone writes it, but in terms of me personally doing the designing + coding, it fall pretty low on the list so would be a long time in the future.

**Reply by dmitry-erin** [reply]
Hi!
I made some changes, they could be interesting to you. Not sure, that the approach is "well", cause it doesn't change the overall logic, but just substitute the colors in a few code snippets.
https://github.com/labwc/labwc/commit/28391b2d2609346d23f7ec4fdbd74cbf8d7d3322
I would be grateful for getting your comments.

**Reply by johanmalm** [reply]
Thanks for working on this.
Is there any chance you could submit it as a Pull Request to get some more eyes on it?
I like it in principle, but just need to get my head around the implementation.
I haven't it in my mind that we'd need to have multiple `theme->corner_top_right_active_normal` etc. Is that not the case?

**Reply by dmitry-erin** [reply]
Thanks for the comments.
I created PR https://github.com/labwc/labwc/pull/1431
Not sure about `theme->corner_top_right_active_normal`, need to check.
